### PR TITLE
#139284031 add digital products

### PR DESCRIPTION
**Pull request by boswellgathu**
#### What does this PR do?
Adds ability to sell digital products, e.g, Music, Videos, Software and Books
 #### Description of Task to be completed?
A vendor should be able to add digital products to the shop
A user should be able to buy digital products from the shop
 #### How should this be manually tested?
As a vendor check the digital button while adding a product
 #### What are the relevant pivotal tracker stories?
#139284031
 #### Screenshots (if appropriate)
screen shot 2017-04-01 at 15 44 04
screen shot 2017-04-01 at 15 46 01
screen shot 2017-04-01 at 15 50 36
screen shot 2017-04-01 at 15 54 28

 #### Questions:

### CODE REVIEW

**Comment by gngchrs** [comment]
@andela-Bgathu reopen when you have the correct screenshots, and a working/passing circle CI.

**Review by jkanyua — APPROVED** [review]

**Comment by jkanyua** [comment]
Rebuild CI without cache and see if tests pass @andela-Bgathu

**Comment by jkanyua** [comment]
@andela-Bgathu Never mind, tried that. Didn't work

File: imports/plugins/included/product-detail-simple/client/components/productDetail.js [hunk 1]
```diff
@@ -74,6 +94,117 @@ class ProductDetail extends Component {
...
+
+  renderDigitalDetails() {
+    if (this.props.hasAdminPermission && this.state.digital) {
+      return (
+        <div>
+          <Divider
+            label="Digital product options"
+          />
+          <FieldGroup
+            componentClass="select"
```
Comment (gngchrs) [inline imports/plugins/included/product-detail-simple/client/components/productDetail.js:188]: These categories are hard coded. What happens when we want to a new digital product category? Say Gift Cards, we'll have to do a new build of the whole project. 

Please think of this from that perspective. Closing for now.

Comment (gngchrs) [reply imports/plugins/included/product-detail-simple/client/components/productDetail.js:188]: Use the same approach @andela-jkanyua used for static pages, they are static but dynamically generated.

**Pull request closed without merging**